include/nighthawk/client/output_collector.h

```diff
@@ -26,7 +29,8 @@ class OutputCollector {
    */
   virtual void addResult(absl::string_view name, const std::vector<StatisticPtr>& statistics,
                          const std::map<std::string, uint64_t>& counters,
-                         const std::chrono::nanoseconds execution_duration) PURE;
+                         const std::chrono::nanoseconds execution_duration,
+                         const absl::optional<Envoy::SystemTime>& first_acquisition_time) PURE;
```

Comment:
will likely be addressed when the comment is added, but I'm not sure i understand what acquisition means in this context.